omnibus/config/software/datadog-agent.rb

```diff
@@ -84,6 +84,9 @@
   if windows?
     platform = windows_arch_i386? ? "x86" : "x64"
     command "invoke customaction.build --arch=" + platform
+    unless windows_arch_i386?
+      command "invoke installcmd.build --arch=" + platform
```

Comment:
Does this change anything for the regular (non-cf) Windows build?

Reply:
It will generate the binary, but where it's placed, it shouldn't be included in the final MSI